frontend/static/js/pages/lobbys/lobby.js

```diff
@@ -263,6 +263,17 @@ var app = new Vue({
             link.href = canvas.toDataURL("image/png");
             link.click();
         },
+
+        async leaveLobby() {
+            event.preventDefault();
+            const resp = await fetchJson(`/api/lobbys/leave/${LOBBY_ID}`, "DELETE");
+            if (resp.status > 399) {
+                alert("Unable to leave lobby. Check that you are not the Owner")
+            }                
+            else {
+                window.location.href = '/';
+            }
```

Comment:
We can make this more specific

```suggestion
            if (resp.status == 400) {
                alert("Unable to leave lobby. Check that you are not the owner")
            } else if (resp.status > 400) {
                alert("Unable to leave lobby. Please try again or let us know if there is something broken.")
            } else {
                window.location.href = '/';
            }
```